specification/compute/resource-manager/readme.md

```diff
@@ -219,8 +219,7 @@ input-file:
 - Microsoft.Compute/stable/2020-06-01/runCommands.json
 - Microsoft.Compute/stable/2019-04-01/skus.json
 - Microsoft.Compute/stable/2020-06-30/disk.json
-- Microsoft.Compute/preview/2020-09-30/gallery.json
-- Microsoft.Compute/preview/2020-09-30/sharedGallery.json
+- Microsoft.Compute/stable/2019-12-01/gallery.json
 - Microsoft.ContainerService/stable/2017-01-31/containerService.json
 - Microsoft.Compute/preview/2020-10-01-preview/cloudService.json
```

Comment:
Should this preview CloudService also be removed?

Reply:
Is there any reason to remove it too? I think here is only discuss impact to change shareGallery.json ? maybe @msyyc or @yungezz to have more context to answer this

Reply:
@akning-ms I think it's removed because it's a preview version

Reply:
If so, I think we don't need remove it if it is already in previous SDK version